src/to_vec.jl

```diff
@@ -61,25 +61,17 @@ function to_vec(X::Diagonal)
 end
 
 function to_vec(X::Transpose)
-
-    x_vec, x_from_vec = to_vec(X.parent)
-
     function Transpose_from_vec(x_vec)
-        return Transpose(x_from_vec(x_vec))
+        return Transpose(permutedims(reshape(x_vec, size(X))))
```

Comment:
Why did this have to be reverted?

Reply:
It turned out that if you just look at the `parent` field, `first(to_vec(X)) == first(to_vec(X'))`, which as it turns out, is bad. The moral of the story was not to look at private fields, and to take the semantics of the matrix seriously.